<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_aziz</class>
 <widget class="QMainWindow" name="gestion_aziz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-color: rgb(230, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);
background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-image: url(:/new/prefix1/img/maxresdefault.jpg);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Clients</string>
     </attribute>
     <widget class="QLineEdit" name="rech_client">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_client">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(213, 78, 33);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Gestion Clients&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="client_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Naissance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ajout</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cartes Fidelite</string>
     </attribute>
     <widget class="QTableView" name="table_carte">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/prefix1/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Gestion Cartes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_carte">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="carte_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Client</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Points</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Remise</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ajout</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_client"/>
   <addaction name="actiondelete_client"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionadd_carte"/>
   <addaction name="actiondelete_carte"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionadd_points"/>
   <addaction name="separator"/>
   <addaction name="actionget_remise"/>
   <addaction name="separator"/>
   <addaction name="actiondeconnecter"/>
  </widget>
  <action name="actionadd_client">
   <property name="icon">
    <iconset>
     <normaloff>img/user-add-icon---shine-set-add-new-user-add-user-30.png</normaloff>
     <normalon>:/new/prefix1/img/user-add-icon---shine-set-add-new-user-add-user-30.png</normalon>img/user-add-icon---shine-set-add-new-user-add-user-30.png</iconset>
   </property>
   <property name="text">
    <string>add_client</string>
   </property>
  </action>
  <action name="actiondelete_client">
   <property name="icon">
    <iconset>
     <normaloff>img/images.png</normaloff>
     <normalon>:/new/prefix1/img/images.png</normalon>img/images.png</iconset>
   </property>
   <property name="text">
    <string>delete_client</string>
   </property>
  </action>
  <action name="actionadd_carte">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/Add-icon.png</normaloff>
     <normalon>:/res/img/Add-icon.png</normalon>:/new/prefix1/img/Add-icon.png</iconset>
   </property>
   <property name="text">
    <string>add_carte</string>
   </property>
  </action>
  <action name="actiondelete_carte">
   <property name="icon">
    <iconset>
     <normaloff>img/delete-icon.png</normaloff>
     <normalon>:/new/prefix1/img/delete-icon.png</normalon>img/delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>delete_carte</string>
   </property>
  </action>
  <action name="actionadd_points">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/img/archiver.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>add_points</string>
   </property>
  </action>
  <action name="actionget_remise">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/img/ajouter article.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>get_remise</string>
   </property>
  </action>
  <action name="actiondeconnecter">
   <property name="icon">
    <iconset>
     <normaloff>img/Windows-Log-Off-icon.png</normaloff>
     <normalon>:/new/prefix1/img/Windows-Log-Off-icon.png</normalon>img/Windows-Log-Off-icon.png</iconset>
   </property>
   <property name="text">
    <string>Se deconnecter</string>
   </property>
   <property name="toolTip">
    <string>deconnecter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
